client/views/room/Header/Header.js

```diff
@@ -64,7 +66,7 @@ const DirectRoomHeader = ({ room }) => {
 const RoomHeader = ({ room, topic }) => {
 	const { isMobile } = useLayout();
 	const avatar = <RoomAvatar room={room}/>;
-
+	const { t } = room;
```

Comment:
we have a type testing on the default component, please create your own variation and apply the test there